### DISCUSSION TITLE
Include cash balances on overview page

### DISCUSSION THREAD

**Discussion by GrantBirki** [ghostfolio/ghostfolio#2361] [Ideas]
# Include cash balances on overview page

I am purposing that cash balances should be included on the main overview page, aka `/home`

It feels rather odd that this page wouldn't include cash balances. I see this page as a "high level **overview**" of all the assets in a person's account and that should include cash accounts. This should be a "total net worth view".

The page in question:

![Image](https://github.com/ghostfolio/ghostfolio/assets/23362539/947c11a8-38a6-4f76-a061-653c70489f37)

**Comment by dtslvr** [comment]
Duplicate of https://github.com/ghostfolio/ghostfolio/discussions/2308.

**Reply by GrantBirki** [reply]
@dtslvr any updates on this? I have been patiently waiting for a few months now to see if this could become a reality. I even started work on my own fork to include this feature but I am not familiar enough with this project to make a PR with the change. This would be greatly appreciated by myself and many others!

> PS it could even be a setting that admins (or users) could toggle if you wanted to keep the current functionality but give users the option to include cash balances.

Thanks! ❤️
